.github/workflows/test.yaml

```
@@ -106,6 +106,40 @@ jobs:
         name: coverage.unit.out
         path: coverage.unit.out
 
+  integration-tests-enterprise-postgres:
+    environment: "Configure ci"
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.16'
```

We're using 1.17 elsewhere now as of #1890:

```suggestion
        go-version: '^1.17'
```